airbyte-integrations/connectors/destination-s3/build.gradle

```diff
@@ -27,12 +27,12 @@ application {
 dependencies {
 
     // csv
-    implementation 'com.amazonaws:aws-java-sdk-s3:1.11.978'
+    implementation 'com.amazonaws:aws-java-sdk-s3:1.12.261'
```

Comment:
how did you get 12.261? That was released over 2 years ago according to https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-s3
Last version is 12.770

Reply:
there's other places where we're still using 261, I think

Reply:
Updated the version to 12.770 for the redshift destination also